src/js/engagement_view/src/components/SagemakerNotebook.tsx

```diff
@@ -0,0 +1,29 @@
+import { getEngagementEdge } from "../modules/GraphViz/engagement_edge/getApiURLs";
+
+const notebookEdge = getEngagementEdge();
+
+
+export const getNotebookUrl = async (): Promise<boolean | null> => {
```

Comment:
doesn't really need to be a boolean

Reply:
`export const getNotebookUrl = async (): Promise<> => {`
I think the type should be this, and then we should not `return true` lower down.

Reply:
Changed return type.